editor/modes/visual-editor/block.js

```diff
@@ -122,6 +131,13 @@ export default connect(
 				hovered: false,
 				uid: ownProps.uid
 			} );
+		},
+		onFocus( config = {} ) {
```

Comment:
Should we be assigning the default config into state memory or would it be more appropriate to apply the default in `mapStateToProps`?